src/nanoemoji/write_fea.py

```diff
@@ -18,33 +18,25 @@
 
 
 from absl import app
+from absl import flags
 from nanoemoji import codepoints
-from nanoemoji.glyph import glyph_name
+from nanoemoji import features
```

Comment:
I had to move `generate_fea` function to a separate `nanoemoji.features` submodule, otherwise defining a flag named `output_file` inside `nanoemoji.write_fea` (the CLI entry point) would create a duplicate flag conflict (as it's also defined in the main `nanoemoji.config` module; the `test_helpers.py` where importing from `nanoemoji.write_fea` and absl.flags was complaining). I think it's neater to keep the importable core logic separate from the modules that are meant to be executed from the console.